application/cms/page_service.py

```diff
@@ -63,6 +65,12 @@ def create_page(self, page_type, parent, data, created_by, data_source_forms, ve
             position=len([c for c in parent.children if c.latest]),
         )
 
+        measure = Measure(uri=uri)
+        measure.subtopics = [Subtopic.query.filter_by(uri=parent.uri, position=parent.position).one()]
```

Comment:
Don't we need to set the `position` here to be 1 more than the position of the last current measure within that topic?

Reply:
👍